Make startup config optional (#132)
eclipse-ankaios/ankaios: doc/docs/usage/quickstart.md

[here](installation.md). The following examples assumes that the
installation script has been used with default options.

> You can start workloads in Ankaios in several ways.
> For example you may define a file with the startup configuration and start Ankaios with systemd.
> The startup configuration contains all the workloads and their
> configuration which should be started when Ankaios starts up.

Reviewer: ```suggestion
You can start workloads in Ankaios in a number of ways.
For example, you can define a file with the startup configuration and use systemd to start Ankaios.
The startup configuration file contains all of the workloads and their configuration that you want to be started when Ankaios is started.
```
